src/call-for-proposals/why-you-should-submit-proposal.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: A conference proposal? Me? Why?
+---
+
+<style>
+  .box_cfp figure {
+    float: right;
+    max-width: 250px;
+    margin-left: 1em;
+    margin-bottom: 1em;
+  }
+
+  .box_cfp h3 {
+    margin-top: 0;
+    margin-bottom: 0.5em;
+  }
+
+  .box_cfp p:nth-child(3) {
+    margin-top: 0.5em;
+  }
+
+  h2 {
+    margin-top: 3em;
+  }
+</style>
+
+<div class="box box_yellow">
+  <h3>Everyone has something to share</h3>
+  <p>You might be a world-class expert or you might be a new beginner. Either way, you almost certainly have <strong>something worth sharing</strong> with other people.</p>
+  <p>It could be a story, a lesson, an idea, an experience, a protest, a discovery, a failure, a creation, a work in progress, a sucess, a challenge- <strong>anything at all.</strong></p>
+  <p>The perspectives and experiences of new contributors are <strong>just as valuable as</strong> (and sometimes more surprising than) those of seasoned contributors.</p>
+</div>
+
+<div class="box box_blue">
+  <h3>Pycon UK has a large and enthusiastic audience</h3>
+  <p>Whatever it is you have to share, <strong>it's going to strike a chord</strong> with at least someone in our audience. Your contribution, however modest or specialised you think it might be, could be exactly what someone else has been waiting for.</p>
+  <p>A session at PyCon UK is an excellent way to <strong>raise interest in a project</strong> or <strong>find new collaborators</strong>, or to <strong>draw attention to a problem</strong>.</p>
```

Comment:
nitpick: too many 'or's and 'to's

Instead:

<p>A session at PyCon UK is an excellent way to <strong>raise interest in a project</strong>, <strong>find new collaborators</strong>, or <strong>draw attention to a problem</strong>.</p>

Reply:
Updated.